### PR TITLE
Add links to naga wgsl spec milestone

### PR DESCRIPTION
**Pull request by 9SMTM6**
**Checklist**

- [x] Run `cargo clippy`. (not applicable)
- [x] Run `RUSTFLAGS=--cfg=web_sys_unstable_apis cargo clippy --target wasm32-unknown-unknown` if applicable. (not applicable)
- [ ] Add change to CHANGELOG.md. See simple instructions inside file.
- [ ] Check that nothing wrong gets implied

**Connections**
https://github.com/gfx-rs/wgpu/issues/2994

**Description**
Adds a semi prominent section describing WGSL evolution to make it easier for people to track whether their issues may stem from it using an as of yet unsupported feature.

**Testing**
I wish I knew a good way to test documentation besides asking people or just throwing it out there

### CODE REVIEW

**Comment by jimblandy** [comment]
I would prefer to land #3009 instead of this. That provides the same links, but I think it's a clearer explanation of the status quo.

**Comment by jimblandy** [comment]
#3009 has landed instead.

**Comment by 9SMTM6** [comment]
I was tired when I wrote this, without question your MR is superior. You just merged it with a reference to your own repo in a link, you probably want to update it to contain a markdown relative link (?), like I did in this MR.

File: README.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ Both [HLSL](https://github.com/Microsoft/DirectXShaderCompiler) and [GLSL](https
 have compilers to target SPIR-V. All of these shader languages can be used with any backend, we
 will handle all of the conversion. Additionally, support for these shader inputs is not going away.
 
+The WGSL Spec is still in developement and full conformity with the current spec is unlikely. 
+[Below](#tracking-the-evolving-wgsl-spec) we elaborate a bit more.
```
Comment (9SMTM6) [inline README.md:88]: Specifically here

Comment (jimblandy) [reply README.md:88]: You're right - thanks. Fixed in 94ce7639.

**Pull request closed without merging**